test/parallel/test-net-autoselectfamily-commandline-option.js

```diff
@@ -85,6 +85,8 @@ function createDnsServer(ipv6Addr, ipv4Addr, cb) {
         if (common.hasIPv6) {
           assert.strictEqual(error.code, 'ECONNREFUSED');
           assert.strictEqual(error.message, `connect ECONNREFUSED ::1:${port}`);
+        } else if (error.code === 'EAFNOSUPPORT') {
+          assert.strictEqual(error.message, `connect EAFNOSUPPORT ::1:${port} - Local (undefined:undefined)`);
```

Comment:
That `undefined:undefined` is a bit weird.

Reply:
Well, it makes sense for systems without IPv6 support to not know how to represent invalid/unknown source `[address]:port`. 😄
It is possible to adjust all `exceptionWithHostPort(..., additional)` calls so they will try to provide more descriptive error in this specific case, but not worth it imho.